README.md

```diff
@@ -1,3 +1,119 @@
-# Webex Components
+<h1 align='center' style='border-bottom: none;'>Webex Components</h1>
+<h3 align='center'>Library of React components to easily embed into your web applications!</h3>
+<p align='center'>
+<a href='https://circleci.com/gh/webex/components'>
+    <img alt='CircleCI' src='https://circleci.com/gh/webex/components.svg?style=shield'>
+  </a>
+  <a href='https://www.npmjs.com/package/@webex/components'>
+    <img alt='npm latest version' src='https://img.shields.io/npm/v/@webex/components?label=npm%40latest'>
+  </a>
+  <a href='#badge'>
+    <img alt='semantic-release' src='https://img.shields.io/badge/%20%20%F0%9F%93%A6%F0%9F%9A%80-semantic--release-e10079.svg'>
+  </a>
+  <a href='https://github.com/webex/components/blob/master/package.json#L28'>
+    <img src='https://img.shields.io/npm/l/webex.svg' alt='license'>
+  </a>
+</p>
 
-The library for Webex Components.
+**Webex Components** is a set of [React](https://reactjs.org) components following Webex standard styling, aimed at react developers that want to embed the components into their applications.
+
+
+## Table of Contents
+- [Install](#install)
+- [Usage](#usage)
+- [Contributing](#contributing)
+- [License](#license)
+- [Support](#support)
+- [Team](#team)
+
+## Install
+
+```bash
+npm i -S @webex/components @momentum-ui/react
+```
+
+## Usage
+**Webex Components** are built with [Momentum UI](https://github.com/momentum-design/momentum-ui) React components. We need to externally import all the fonts, icons, images and core *CSS* manually. There are two ways to do this:
+
+### 1. CSS
+1. Copy the `@momentum-ui/core/fonts` directory to the 'fonts' directory for your site.
+2. Copy the `@momentum-ui/core/images` directory to the 'images' directory for your site.
+3. Copy the `@momentum-ui/core/css/momentum-ui.min.css` to your styles/css directory.
+4. In the `<head>`, of your HTML reference the location of your momentum-ui.min.css:
+
+    `<link rel='stylesheet' href='path/to/momentum-ui/css/momentum-ui.min.css'>`
+
+***
+### 2. SCSS
+1. Copy the `@momentum-ui/icons/fonts` directory to the `fonts` directory for your site.
+2. Add an `$brand-font-folder`, `$icon-font-path` variables to your app's variables.scss file.
```

Comment:
```suggestion
2. Add `$brand-font-folder` and `$icon-font-path` variables to the top of your app's sass file.
```

Since we don't really refer to `variables.scss` anywhere.